### ISSUE TITLE
Vid2Player: Controllable Video Sprites that Behave and Appear like Professional Tennis Players

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#46]
# TL;DR

Research to synthesize tennis matches in a realistic way. Based on the domain knowledge of tennis, they divided the player's movement into two parts, “catching up and hitting the ball" and "taking the next position”.  They select the nearest frames from them to synthesize. There are no shadows, so it can be identified as a synthesized, but the movements themselves are quite natural.
![image](https://user-images.githubusercontent.com/24524018/90498846-c75dcc80-e183-11ea-91ac-85ff92ffc833.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.04524

# Submission Dates(yyyy/mm/dd)
2020/08/11

# Authors and institutions
Haotian Zhang, Cristobal Sciutto, Maneesh Agrawala, Kayvon Fatahalian
- Stanford University

# Methods

# Results

# Comments
project page : 
https://cs.stanford.edu/~haotianz/research/vid2player/
